generator.rb

```diff
@@ -0,0 +1,99 @@
+# Checks if letter is a vowel
+def is_vowel?(letter)
+  if ["a", "e", "i", "o", "u"].include?(letter.downcase)
+    return true
+  else
+    return false
+  end
+end
+
+# Returns an array of index locations of vowels in a word
+def get_vowel_index(word)
+  vowel_index = []
+  word.each_char.with_index do |char, index|
+    if is_vowel?(char)
+      vowel_index << index
```

Comment:
This solution is effective, but it does a lot more work than it needs to. You could more easily iterate forward or backwards thru the array and return the index of the first vowel.